### DISCUSSION TITLE
Me when i get invited

### DISCUSSION THREAD

**Discussion by Theoyeah** [DEMOLITIONDON96/Demolition-Engine#7] [General (Memes)]
![image](https://user-images.githubusercontent.com/97792861/162253964-5b91e6c6-6155-4242-9b77-1ed8dab5e5fc.png)

**Comment by ThatOneSillyGuyDon** [comment]
y e s .

**Comment by Theoyeah** [comment]
WWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWEEEEEEEEEEEEEEEEEEEEEE

**Comment by Blossomical** [comment]
um do u even know source, i see u in discussions coding with lua only

**Reply by Theoyeah** [reply]
yes i do know how make some source things

**Reply by Theoyeah** [reply]
i ask lua sometimes bc im lazy somtimes

**Comment by megasssdev** [comment]
![image](https://user-images.githubusercontent.com/61982239/162426768-115874ba-5118-4143-bbf6-464b0de922de.png)

**Reply by ThatOneSillyGuyDon** [reply]
> ![image](https://user-images.githubusercontent.com/61982239/162426768-115874ba-5118-4143-bbf6-464b0de922de.png)

tbh, this engine was literally made specifically to add features the community wanted most, but got rejected by the Psych Team

**Reply by megasssdev** [reply]
> tbh, this engine was literally made specifically to add features the community wanted most, but got rejected by the Psych Team

yea thats actually true

**Reply by Theoyeah** [reply]
ye
